src/python/pants/backend/go/lint/gofmt/subsystem.py

```diff
@@ -1,8 +1,11 @@
 # Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-from pants.option.option_types import SkipOption
+from pants.option.option_types import ArgsListOption, SkipOption
 from pants.option.subsystem import Subsystem
+from pants.util.ordered_set import FrozenOrderedSet
+
+SUPPORTED_GOFMT_ARGS = FrozenOrderedSet(("-e", "-r", "-s"))
```

Comment:
Python's `frozenset` should be sufficient here.

Reply:
Sorry, I should have mentioned that. The reason I've used an ordered one is to be able to confirm that this thing appears in the exception message when running a test with invalid arguments passed.

Reply:
:+1:

Reply:
I also wanted the help message to be consistent and output the arguments in the same order all the time.